aas_core_meta/v3.py

```diff
@@ -1,5 +1,11 @@
 """
-Provide the meta-model for Asset Administration Shell V3.0.
+The presented version of the Metamodel within this documentation
+is related to the work of aas-core-works, which can be found here.
```

Comment:
Add Link to aas-core-works

Reply:
See 59c1702a0ef8809115c3eaa8823a0a1dc3435726